src/View/Helper.php

```diff
@@ -197,7 +197,9 @@ protected function _confirm($message, $okCode, $cancelCode = '', $options = [])
      */
     public function addClass(array $options = [], $class = null, $key = 'class')
     {
-        if (isset($options[$key]) && trim($options[$key])) {
+        if (is_array($options[$key])) {
```

Comment:
I think you need an isset() here as well.

Reply:
Yeah, it was a quick patch - I'll have a look at writing a test.

Thank you for offering help - if I get stuck, I'll ask! :)

Reply:
Thanks.